docs/ref/contrib/postgres/indexes.txt

```diff
@@ -24,3 +24,25 @@ available from the ``django.contrib.postgres.indexes`` module.
     PostgreSQL. You can install it using the
     :class:`~django.contrib.postgres.operations.BtreeGinExtension` migration
     operation.
+
+``BRINIndex``
+=============
+
+.. class:: BRINIndex(pages_per_range=None)
+
+    Creates a `BRIN index
+    <https://www.postgresql.org/docs/devel/static/brin-intro.html>`_. It is not
+    an optimal index to use unless your table is very large.  The `page inspect
```

Comment:
I'm not sure if the Django docs is a good place to discuss the details of the index. Linking to the PostgreSQL docs seems better.